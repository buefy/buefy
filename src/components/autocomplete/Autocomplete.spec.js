import { shallowMount } from '@vue/test-utils'
import BAutocomplete from '@components/autocomplete/Autocomplete'

describe('BAutocomplete', () => {
    it('is called', () => {
        const wrapper = shallowMount(BAutocomplete)
        expect(wrapper.name()).toBe('BAutocomplete')
        expect(wrapper.isVueInstance()).toBeTruthy()
    })
})
