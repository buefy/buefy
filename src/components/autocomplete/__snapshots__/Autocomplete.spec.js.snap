// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BAutocomplete render correctly 1`] = `
<div class="autocomplete control">
    <div class="control is-clearfix"><input type="text" autocomplete="off" aria-autocomplete="list" class="input">
        <!---->
        <!---->
        <!---->
    </div>
    <div class="dropdown-menu" style="display: none;" name="fade">
        <div class="dropdown-content" style="display: none;">
            <!---->
            <!---->
            <!---->
            <!---->
        </div>
    </div>
</div>
`;
